code/MediaWikiTest/MediaWikiTest/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace MediaWikiTest
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string language = "fr";
+            string text = string.Empty;
+            
+            text = fullSearch(string.Join("", args), language).Result;
+            
+            Console.WriteLine(text);
+            Console.ReadKey();
+        }
+
+        static async Task<string> fullSearch(string term, string language){
+            string text = await GetData(term, language);
+           
+                if(string.IsNullOrEmpty(text)){
```

Comment:
You can possibly refactor this with a optional parameter with the default language Englisch.

Reply:
In this function, the `language` parameter shouldn't get a default value.